php/libraries/TimePoint.class.inc

```diff
@@ -208,7 +209,8 @@ class TimePoint
         int $SubprojectID,
         ?string $nextVisitLabel,
         \Site $psc,
-        \Project $project
+        \Project $project,
+        int $lang = 1
```

Comment:
This will break https://github.com/aces/Loris/blob/master/modules/api/php/endpoints/candidate/visit/visit.class.inc#L273

Reply:
@xlecours even with the default?

Reply:
@jesscall good point. resolved :)